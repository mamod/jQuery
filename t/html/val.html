<!DOCTYPE html>
<html>
<head>
  <style>
  body { color:blue; }
  </style>
  
</head>
<body>
  <select id="single">
    <option>Single</option>
    <option>Single2</option>
    <option>Single3</option>
  </select>

  <select id="multiple" multiple="multiple">
    <option selected="selected">Multiple</option>
    <option>Multiple2</option>
    <option selected="selected">Multiple3</option>
  </select><br/>
  <input type="checkbox" name="checkboxname" value="check1"/> check1
  <input type="checkbox" name="checkboxname" value="check2"/> check2
  <input type="radio"  name="r" value="radio1"/> radio1
  <input type="radio"  name="r" value="radio2"/> radio2
  
  
  <input id="test" type="test" value="sssss" />
  
<script>
    
    

</script>

</body>
</html>
