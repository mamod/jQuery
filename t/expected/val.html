<!DOCTYPE html>
<html>
<head>
  <style>
  body { color:blue; }
  </style>
  
</head>
<body>
  <select id="single">
    <option>Single</option>
    <option selected>Single2</option>
    <option>Single3</option>
  </select>

  <select id="multiple" multiple>
    <option>Multiple</option>
    <option selected>Multiple2</option>
    <option selected>Multiple3</option>
  </select><br>
  <input type="checkbox" name="checkboxname" value="check1" checked> check1
  <input type="checkbox" name="checkboxname" value="check2" checked> check2
  <input type="radio" name="r" value="radio1" checked> radio1
  <input type="radio" name="r" value="radio2"> radio2
  
  
  <input id="test" type="test" value="sssss">
  
<script>
    
    

</script>

</body>
</html>
